pkg/prometheus/promcfg.go

```diff
@@ -41,6 +41,7 @@ const (
 	kubernetesSDRoleEndpointSlice = "endpointslice"
 	kubernetesSDRolePod           = "pod"
 	kubernetesSDRoleIngress       = "ingress"
+	kubernetesSDRoleNode          = "node"
```

Comment:
My idea was to use this but I am letting specify role in ScrapeConfig spec. I will revisit this